app/controllers/candidate_interface/sign_in_controller.rb

```diff
@@ -96,5 +101,28 @@ def create_from_expired_token
     def candidate_params
       params.require(:candidate).permit(:email_address)
     end
+
+    def send_candidate_sign_in_confirmation_email
+      return if cookies.signed[:sign_in_confirmation] == @local_user.id
+
+      cookies.signed[:sign_in_confirmation] = {
+        value: @local_user.id,
+        expires: 20.years.from_now,
```

Comment:
I think this should be 6 months right? https://www.apply-for-teacher-training.service.gov.uk/candidate/cookies

Reply:
I think so, am a little uncertain... it would need to be the same for the providers too in that case.

Surely if we set this cookie to expire in six months, this will trigger a new sign in email on a log in after this date is exceeded for both providers and candidates... is this what we want to happen?